kura/org.eclipse.kura.rest.identity.provider/src/main/java/org/eclipse/kura/internal/rest/identity/provider/IdentityRestServiceV2.java

```diff
@@ -195,6 +195,12 @@ public IdentityConfigurationDTO getIdentityDefaultByName(
     public Response deleteIdentity(final IdentityDTO identity) {
         logger.debug(DEBUG_MESSAGE, "deleteIdentity");
         try {
+
+            if (identity.getName() == null) {
```

Comment:
```suggestion
            if (isNull(identity.getName())) {
```
More readable like this?
If so, you'll need to add a static import